allensdk/brain_observatory/behavior/data_objects/eye_tracking/eye_tracking_table.py

```diff
@@ -236,6 +249,62 @@ def from_data_file(
 
         return EyeTrackingTable(eye_tracking=eye_tracking_data)
 
+    @classmethod
+    def _validate_frame_time_alignment(
+            cls,
+            frames: np.ndarray,
+            times: StimulusTimestamps,
+            is_metadata_frame_present: bool = False
+    ) -> Tuple[np.ndarray, StimulusTimestamps]:
+        """
+        Checks whether frames or timestamps need to be modified in order to be
+            aligned with each other. If so, does the alignment
+
+        Parameters
+        ----------
+        frames: eye tracking frames
+        times: eye tracking timestamps
+        is_metadata_frame_present: Whether frames contains a metadata frame as
+            the first frame
+
+        Returns
+        -------
+        Tuple of frames, timestamps, where frames and timestamps have been
+            corrected to be aligned with each other
+        """
+        if len(times.value) > len(frames):
+            # It's possible for there to be more timestamps than frames in a
+            # case of non-transferred frames/aborted frames
+            # See discussion in https://app.zenhub.com/workspaces/allensdk-10-5c17f74db59cfb36f158db8c/issues/alleninstitute/allensdk/2376 # noqa
```

Comment:
Should refer to github issue directly not via zenhub. My understanding  that not everyone has a access to zenhub.